### PR TITLE
Extend snippet doc with better exmaple

### PR DESCRIPTION
**Pull request by TheCadien**
#### What's in this PR?

Extend the snippet doc for more Background

#### Why?

Missing doc and fix a Typo in Doc

### CODE REVIEW

File: bundles/snippet.rst [hunk 1]
```diff
@@ -75,6 +101,31 @@ Properties
...
+                <title lang="en">Footer Social Media</title>
+            </meta>
+            <params>
+                <param name="default" value="social_media"/>
+            </params>
+        </property>
+
+.. note::
+
+    With Sulu 2.1 its possible to use a "Single Snippet Selection" to link specially one snippet!
```
Comment (TheCadien) [inline bundles/snippet.rst]: Think if the PR would accepted, this need for the 2.1 branch should be removed and the line above it should be extended with the "Single_Snippet_Selection".

Comment (niklasnatter) [reply bundles/snippet.rst]: we decided to update the documentation of maintained versions only. at the moment maintain 2.1 and 2.2, so we dont need to backport this for 2.0. would you mind to change this PR to target 2.1 and update this in the PR? 🙂

Comment (TheCadien) [reply bundles/snippet.rst]: @nnatter done :)

**Comment by TheCadien** [comment]
@nnatter change base to 2.1

**Comment by niklasnatter** [comment]
Thank you! 🙂

**Pull request merged**